src/infiniteGrid.js

```diff
@@ -406,6 +406,10 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 		 * @return {eg.InfiniteGrid} instance of itself<ko>자신의 인스턴스</ko>
 		 */
 		setStatus: function(status) {
+			if (!status || !status.cssText || !status.html ||
+				!status.core || !status.data) {
+				return this;
```

Comment:
@happyhj this is just defense code.
generally, the egjs and jquery ignore this situation.